frontend/packages/data-portal/app/components/PageHeader.tsx

```diff
@@ -37,140 +37,146 @@ export function PageHeader({
 
   return (
     <div className="flex flex-auto justify-center">
-      <header
-        className={cns(
-          'flex flex-col items-center',
-          'w-full min-h-[48px] max-w-content',
-        )}
-      >
-        <div className="flex flex-col justify-start w-full">
+      <header className="flex flex-col items-center w-full min-h-[48px]">
+        <div className="flex flex-col justify-start w-full pb-sds-xl">
           <div
             className={cns(
-              // create grid with fit content 1st row / 2nd col
-              'grid grid-cols-[1fr_auto] grid-rows-[1fr_auto]',
-              'w-full',
-              'justify-between',
-              'gap-x-sds-xxl',
-              'px-sds-xl pt-sds-l pb-sds-xxl',
+              'flex justify-center',
+              'border-b-[3px] border-[#d9d9d9]',
```

Comment:
is there a corresponding sds colour for this?

Reply:
I don't really understand your question, might be because I don't have enough knowledge of github and code
1) what is a color token
2) how do I read the code to understand "what is this map to"?

Thanks for clarifying for me!

@codemonkey800